packages/grafana-data/src/transformations/transformers/concat.test.ts

```diff
@@ -0,0 +1,136 @@
+import { toDataFrame } from '../../dataframe/processDataFrame';
+import { concatenateFields, ConcatenateFrameNameMode } from './concat';
+
+export const simpleABC = toDataFrame({
+  name: 'ABC',
+  fields: [
+    { name: 'A', values: [1] },
+    { name: 'B', values: [1, 2] },
+    { name: 'C', values: [1, 2, 3] },
+  ],
```

Comment:
is this a valid configuration? (ie field with not the same lengths)

Reply:
`toDataFrame` will sort that out (making them all the same length), but your comment points out that I am not testing what I though I was.  The two frames should be of different lenghts to make sure they end up the same after concat

Reply:
I updated the test so that ABC is lenght 2, and XYZ is length 3 -- after concat, the total length of all fields is 3